todo_app/app.py

```diff
@@ -8,4 +9,12 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/create-a-todo', methods=['POST'])
+def create_new_todo():
+    new_todo_title = request.form['todo-name']
+    add_item(new_todo_title)
+    return index()
```

Comment:
This works and saves you from code duplication, which is great! However, it leaves the user thinking that this endpoint returns the HTML directly, which sounds like not an issue but can confuse things (e.g. refreshing the page after you've submitted a to-do tries to submit another one or bookmarking the page might lead to errors) - a nicer solution is to pull in the `redirect` function from Flask, and use it like `return redirect('/')`; this tells the users browser to perform the redirect which leads to it understanding the flow better.